CONTRIBUTORS.yaml

```
@@ -1885,3 +1886,22 @@ biont:
     funding_system: cordis
     funding_statement: |
       Co-funded by the European Union
+
+ukri:
+    name: UKRI
+    github: false
+    joined: 2023-10
+    avatar: "https://github.com/galaxyproject/training-material/blob/main/shared/images/ukri.png?raw=true"
+    url: https://elixiruknode.org/projects/elixir-uk-dash/
+    funder: true
+    funding_statement: |
+      UK Research and Innovation (UKRI) is the national funding agency investing in science and research in the UK. Invest in research that enriches lives, drives economic growth, and creates jobs and high-quality public services.
+
+ELIXIR-UK-DaSH:
+    name: DASH UK
+    joined: 2023-10
+    avatar: "https://github.com/galaxyproject/training-material/blob/main/shared/images/dash.png?raw=true"
+    url: https://elixiruknode.org/projects/elixir-uk-dash/
+    funder: true
```

```suggestion
    funder: true
    funding_system: ukri
    funding_id: MR/V038966/1
```

additionally is this the correct funding ID? It seems to also be used on https://training.galaxyproject.org/training-material/hall-of-fame/elixir-fair-data/ and just want to double check.

As of https://github.com/galaxyproject/training-material/pull/4408 we'll have the ability to annotate some more information about funders which could be useful, but we still need to finalise that first.